test/TestServerProjectsLowLevel/httpInfrastructure/src/Generated/HttpRedirectsClient.cs

```diff
@@ -1022,6 +1053,7 @@ public virtual Response Delete307(RequestContent content, RequestContext context
             try
```

Comment:
Looks there is no test case for 308, so is it because 308 is an invalid case or we missed this test case?

Reply:
@lirenhe The upstream `autorest.testserver` doesn't have a 308 case: https://github.com/Azure/autorest.testserver/blob/main/swagger/httpInfrastructure.json